pkgs/development/tools/build-managers/gup/default.nix

```diff
@@ -1,12 +1,12 @@
 { stdenv, fetchFromGitHub, nix-update-source, lib, python3
 , which, runtimeShell, pylint }:
 stdenv.mkDerivation rec {
-  version = "0.9.1";
+  version = "0.9.2";
   src = fetchFromGitHub {
+    hash = "sha256-bV5HauM0xmRI/9Pxp1cYLPLA8PbFvPER2y4mAMmgchs=";
     owner = "timbertson";
     repo = "gup";
     rev = "version-${version}";
-    sha256 = "1wfw46b647rkalwds6547ylzy353b3xlklhcl2xjgj2gihvi30mx";
```

Comment:
hash should stay at this line

Reply:
This moved because I updated to an [SRI hash](https://github.com/NixOS/nixpkgs/pull/336413#discussion_r1740267651), and the update script orders attributes alphabetically (`sha256` -> `hash`). It won't need to change ordering again in the future.

Reply:
It's a one-time change, that doesn't seem worth migrating tools for.

Reply:
That update script are just using 4 packages or so. Maybe we should just drop it.